api.go

```diff
@@ -6,14 +6,36 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
-	"net/url"
-	"strconv"
 
 	"github.com/NebulousLabs/Sia/modules"
 	"github.com/NebulousLabs/Sia/types"
+	"github.com/gorilla/mux"
 )
 
-// Does an arbitrary request to the server referenced by link, returns as a byte array.
+// NewAPIRouter handles any urls that begin with '/api/'
+func (es *ExploreServer) NewAPIRouter() {
```

Comment:
weird to have a function named `NewXXX` that doesn't return anything. Perhaps `initAPIRouter`? Also, why is it exported?